File: apzkr-pzpi-21-6-buravkova-daryna/Task1-Server/BeanBlissAPI/Models/Order.cs
namespace BeanBlissAPI.Models
{
    public class Order
    {
        public int Id { get; set; }
        public DateTime OrderDate { get; set; }
        public string OrderStatus { get; set; } = string.Empty;
        public bool IsPaid { get; set; } = false;
        public decimal Price { get; set; }
        public string PaymentDetail { get; set; } = string.Empty;
        public User User { get; set; }
        public Coffee Coffee { get; set; }
        public Machine Machine { get; set; }
    }
}

File: apzkr-pzpi-21-6-buravkova-daryna/Task4-MobileClient/app/src/main/java/com/example/beanblissmobile/models/Order.kt
package com.example.beanblissmobile.models

data class Order(
    val id: Int,
    val orderDate: String,
    val orderStatus: String,
    val isPaid: Boolean,
    val price: Int,
    val paymentDetail: String,
    val coffeeId: Int,
    var coffeeName: String? = null // Додаємо змінну для назви кави
)
